datafusion/expr/src/expr_fn.rs

```diff
@@ -259,6 +260,23 @@ pub fn cast(expr: Expr, data_type: DataType) -> Expr {
     }
 }
 
+/// Create a cast expression
```

Comment:
Maybe we need add more document to explain the difference between this and  `pub fn cast`

Reply:
I have now remove this function